<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoWidget</class>
 <widget class="QWidget" name="DemoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DemoWidget</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Jingyi ?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAction">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>To-Do</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFrustration">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Frustration ?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAction4Frustration">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>To-Do</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisplay">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tbDisplay">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
